index.js

```diff
@@ -25,71 +31,98 @@
 
 // ***** Question 3 *****
 
+function drinkWater(currentThirstLevel) {
+  console.log("Man I sure am thirsty")
+  currentThirstLevel -= 1
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
+
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    return "same same"
+  }
+  if (num == maybeNum) {
+    return "same same (but different)"
+  }
+  else
+    return "different"
```

Comment:
This will work, but just a couple notes: 

- for if/else statements, it's better to use curly parentheses to separate out each condition for readability/flexibility:

```js
  if (num === maybeNum) {
    return "same same"
  } else if (num == maybeNum) {
    return "same same (but different)"
  } else {
    return "different"
  }
```

- Also, you can also convert the string to a number with `parseInt` to avoid `==`:

```js
} else if (num === parseInt(maybeNum)) {
```